api/src/validators/words.js

```diff
@@ -0,0 +1,38 @@
+const v8n = require('v8n')
+
+const validateDict = v8n().passesAnyOf(
+  v8n().undefined(),
+  v8n().exact('wordnet'),
+  v8n().equal(''),
+)
+
+const isString = v8n().string()
+
+const validateGetSingleWord = (req, res, next) => {
+  const { word } = req.params
```

Comment:
`word` is from URL so it's always a string, so no need to check for it 😄

Reply:
removed